styles/_body.scss

```diff
@@ -0,0 +1,47 @@
+.Description {
+    font-family: "Akaya Telivigala";
+    font-weight: 100;
+    text-decoration-line:underline;
+    font-size: 24px;
+    text-shadow: 1px 0 0 #000, 0-1px 0 #000, -1px 0 0 #000;
+    color: $SecondaryColor;
+    grid-area: Description;
+    justify-content: center;
+    text-align: center;
+    text-justify: auto;
+    margin: 20px 20px 20px 20px;
+  }
+
+  .Columna-Img {
+    grid-area: Columna-Img;
+    max-height: 200px;
+    max-width: 200px;
+    min-height: 100px;
+    min-width: 100px;
+    width: 200px;
+    height: 200px;
```

Comment:
go with width, or max-width or min width. Not all 3

Mainly it's just width used. same for height